CHANGELOG.md

```diff
@@ -2,8 +2,10 @@
 
 ## master / unreleased
 
+## 0.4.0 / 2019-11-16
+
 * [CHANGE] The frontend component has been refactored to be easier to re-use. When upgrading the frontend, cache entries will be discarded and re-created with the new protobuf schema. #1734
-* [CHANGE] Remove direct DB/API access from the ruler
+* [CHANGE] Removed direct DB/API access from the ruler. #1579
```

Comment:
Could note here this is not expected to have an impact since we think nobody was using it.
Also mention the flag `-ruler.configs.url` which was already deprecated, so people can check if they were using it.
(BTW why hasn't that flag been removed?)